contracts/src/core/EigenDAThresholdRegistry.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Would the `blob version -> {coding rate, reconstruction threshold, confirmation threshold, num chunks}` map be defined here?
cc @mooselumph